extensions/vertx-http/runtime/src/main/java/io/quarkus/devui/runtime/ReportAnIssueJsonRPCService.java

```diff
@@ -0,0 +1,98 @@
+package io.quarkus.devui.runtime;
+
+import java.io.File;
+import java.io.UnsupportedEncodingException;
+import java.net.URL;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+import java.util.Objects;
+
+import jakarta.enterprise.context.ApplicationScoped;
+
+import org.zeroturnaround.exec.ProcessExecutor;
+
+import io.smallrye.common.os.OS;
+import io.vertx.core.json.JsonObject;
+
+@ApplicationScoped
+public class ReportAnIssueJsonRPCService {
+
+    public JsonObject reportBug() {
+        URLBuilder urlBuilder = new URLBuilder(
+                "https://github.com/quarkusio/quarkus/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml");
```

Comment:
could we make this a config property which is overridable by platform the same way as builder images are ?

Reply:
I'm thinking making it a qute template string so it can take in the info if applicable?